nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -1194,12 +1197,6 @@ def build_train_valid_test_datasets(self):
             test_iters * global_batch_size,
         ]
 
-        if self.trainer.limit_val_batches <= 1.0 and isinstance(self.trainer.limit_val_batches, float):
-            train_valid_test_num_samples[
-                1
-            ] = 1  # This is to make sure we only have one epoch on every validation iteration
-
-        mock_dataset = self.cfg.data.get("mock_dataset", False)
```

Comment:
`mock_dataset` is used in the line 1210. It crashed for me with `NameError: name 'mock_dataset' is not defined`

Reply:
My bad this line got removed by mistake. It's added back in the latest commit.